src/services/repo/repo.service.ts

```diff
@@ -142,4 +135,11 @@ export class RepoService {
       }),
     );
   }
+
+  private getRepoMainPath(remote: string, namespace?: string) {
+    if (namespace) {
+      return path.join(this.fetchService.repoCacheFolder, namespace, encodeURIComponent(remote));
```

Comment:
will this long path cause any issue on win ?

Reply:
it cause the issue either way, its broken on windows depending on the repo